pkg/deploy/install.go

```diff
@@ -246,7 +246,11 @@ func (deploy *Deployer) installMeteringClusterRoleBinding() error {
 		}
 		deploy.logger.Infof("Created the metering cluster role binding")
 	} else if err == nil {
-		deploy.logger.Infof("The metering cluster role binding already exists")
+		_, err = deploy.client.RbacV1().ClusterRoleBindings().Update(res)
```

Comment:
Can we take the opportunity to refactor a bad variable naming? `res` -> `clusterRoleBinding`, for instance, seems like `clusterRole` is used in the following function fix up and it makes it more readable.

Reply:
I'd prefer to do that in a different PR as that variable naming is used throughout the package:
```bash
tflannag@localhost operator-metering [update-clusterroles] ▶ grep -rnw pkg/deploy/ -e "res" | wc -l
52
tflannag@localhost operator-metering [update-clusterroles] ▶ 
```

Reply:
ah okay